tictactoe/console.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace tictactoe
+{
+    public class console
```

Comment:
What does this class do?

Reply:
this needs to be deleted commit e74cf8cf0110e01c6b696060b32b2099cc6ddfb8